modules/data.remote/R/call_MODIS.R

```diff
@@ -16,20 +16,21 @@
 ##' @param package_method string value to inform function of which package method to use to download modis data. Either "MODISTools" or "reticulate" (optional)
 ##' 
 ##' depends on a number of Python libraries. sudo -H pip install numpy suds netCDF4 json
+##' depends on the MODISTools package version 1.1.0
```

Comment:
Can you add a version spec to the DESCRIPTION line as well, please?

Currently:
```
Imports:
    MODISTools,
    reticulate,
   ...
```

Change to:
```
Imports:
    MODISTools (>= 1.1.0),
    reticulate,
    ...
```

Reply:
Good catch @infotroph I had asked this separately on slack...given the breaking change from 1 to 1.1

Reply:
added it. But I don't know if >=1.1.0 will end up breaking it. so I made it ==1.1.0 for now.... if that even works

Reply:
`==` works, but for the record I recommend defaulting to `>=` in most cases like this, because:

* Nonbreaking updates are more common than breaking ones, so letting an upgrade Just Work usually saves time and trouble... and you can't usually know which upgrades will be the exceptions until they're released anyway.
* Since R doesn't like having multiple versions of the same package installed at the same time, requiring an exact version means headaches as soon as you install some other package that wants, say, MODISTools >= 1.1.2.